CMakeLists.txt

```diff
@@ -141,7 +153,12 @@ ament_target_dependencies(toolbox_common
   ${dependencies}
 )
 target_link_libraries(toolbox_common kartoSlamToolbox ${Boost_LIBRARIES})
-rosidl_target_interfaces(toolbox_common ${PROJECT_NAME} "rosidl_typesupport_cpp")
+if(${rosidl_cmake_VERSION} VERSION_LESS 2.5.0)
+  rosidl_target_interfaces(toolbox_common ${PROJECT_NAME} "rosidl_typesupport_cpp")
```

Comment:
As well here, are these changes required for Humble/22.04? If so, we don't need to support backwards, there are other branches for that

Reply:
Yes, these changes are for humble/rolling api change. I also removed them.